apps/docs/src/basics/typography.md

```diff
@@ -0,0 +1,350 @@
+# Typography
+
+::: danger
+Still content to create!
+:::
+
+The onyx font system includes three different font families that can be used throughout your application, pending on the usecase. It contains several headline styles as well as paragraph styles and mono space styles.
+
+::: info
+The font families that can be used with onyx are:
+
+- [Source Sans 3](https://fonts.google.com/specimen/Source+Sans+3)
+- [Source Code Pro](https://fonts.google.com/specimen/Source+Code+Pro?query=source)
+- [Lidl Font Pro](https://www.brand.lidl/brandelements/brandelements/view/id/664#headline_2486)
+- [Kaufland Regular](https://google.com)
+  :::
+
+## Brand Typography
+
+The standard font family is defined to be the Source Sans. If you are creating a new onyx project, this will always be the startingpoint. In the initial phase the Source Sans applies to all color themes that are provided by onyx.
```

Comment:
```suggestion
The default font family is defined to be the Source Sans and Source Code Pro (monospace). If you are creating a new onyx project, this will always be included out-of-the-box. In the initial phase, Source Sans applies to all color themes that are provided by onyx.
```